yarn.lock

```diff
@@ -1920,6 +1920,13 @@
   resolved "https://registry.yarnpkg.com/@cosmjs/utils/-/utils-0.31.3.tgz#f97bbfda35ad69e80cd5c7fe0a270cbda16db1ed"
   integrity sha512-VBhAgzrrYdIe0O5IbKRqwszbQa7ZyQLx9nEQuHQ3HUplQW7P44COG/ye2n6AzCudtqxmwdX7nyX8ta1J07GoqA==
 
+"@cspotcode/source-map-support@^0.8.0":
```

Comment:
Shouldn't be any `yarn.lock` changes in this PR I don't think - probably resolved after removing ts-node/ts-node-dev.